msteams-platform/m365-apps/prerequisites.md

```diff
@@ -91,18 +91,33 @@ To install Microsoft 365 Beta Channel applications in your test environment:
 
 ### Mobile
 
-Install the Microsoft 365 app to preview your Teams personal tab app running your physical Android device or Android emulator:
+#### Android
+
+Install the Microsoft 365 app to preview your Teams personal tab app running on your physical Android device or Android emulator:
 
 1. Ensure you're using a Google Play [supported Android device](https://support.google.com/googleplay/answer/1727131).
 1. Launch the **Play Store** on your Android device.
-1. Search for "Office" and select **Office (Microsoft 365)**.
+1. Search for a supported host for running Teams apps:
+    1. For Microsoft 365 app (formerly Office), search office and select *Microsoft 365 app*.
+    1. For Outlook, search outlook and select *Microsoft Outlook*.
 1. Select the **Install** button.
 
     :::image type="content" source="images/office-android-install.png" alt-text="The screenshot is an example that shows the install button for the Office (Microsoft 365) app in Google Play Store.":::
 
 1. Launch the Microsoft 365 app and sign in with your test tenant credentials.
 1. Open your profile **(Me) > Settings** and scroll to the bottom of the menu.
-1. Ensure you use Microsoft 365 app version 16.0.15726.20000 or higher for Android.
+1. Ensure that you use Microsoft 365 app version 16.0.15726.20000 or later and Outlook app version 4.2247.1 (52247812) or later for Android.
+
+#### iOS
+
+You can preview Teams personal tabs running in Outlook and Microsoft 365 app for iOS by installing the prerelease version on the [TestFlight app](https://testflight.apple.com/) on your iOS device.
+
+1. To access the preview version on TestFlight, please reach out to the product group through your CSAM/Account Manager for an Invitation.
+1. Install the TestFlight app from the **App Store** on your testing device of choice.
```

Comment:
Please confirm is it Apple Store or App Store.
We have changed it to App Store.